test/unit/vfocus/mutating.spec.js

```diff
@@ -0,0 +1,129 @@
+var path = require('path');
+var _ = require('lodash');
+var chai = require('chai');
+var expect = chai.expect;
+
+var h = require('virtual-hyperscript');
+var VText = require('vtree/vtext');
+var VFocus = require(path.resolve(process.cwd(), 'src/vfocus'));
+
+
+describe('VFocus - Mutations', function() {
```

Comment:
Since these tests are mutating the tree, would it be better to not use the return values in the tests? And instead use the original tree (that should have been mutated). That way we ensure that they've actually mutated the tree. 

(Immutable VFocus methods would probably be better, but that's not the way these methods work, or are used in the code.)